app/assets/javascripts/cited_documents.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable camelcase */
+/* global CitedDocuments */
+/* global Blacklight */
+
+(function (global) {
+  var CitedDocuments;
+
+  CitedDocuments = {
+    init: function (el) {
+      var $el = $(el);
+      var data = $el.data();
+      data.documentids.forEach(function(documentId) {
```

Comment:
This could be potentially problematic, especially with a reference that cites all Parker items (500+ HTTP calls). I think we could get a away with one query like this:
```
$.getJSON(data.path, {
  q: (sp968bx9690 OR zy353np1659... ),
  format: 'json',
  rows: 1000
}, function (response) {
```